### PR TITLE
Set up multiple components and some tests

### PR DESCRIPTION
**Pull request by HBMattoon**
TODO: 
- there are a few more react components that need to be done
- split up some of my current react components into sub components
- transfer styling over to styled react components instead of css file
- move icons to s3
- add more images to s3 (only have 4 pairs atm)
- **more tests**

### CODE REVIEW

File: client/dist/style.css [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  height: 30;
+
+}
+
+.mybutton {
+  width: 33.3%;
+}
+
+#wishlist {
+  padding-left: 15px;
```
Comment (willresen) [inline client/dist/style.css]: Wishlist and redeem can be combined into one by separating them by a comma. ` #wishlist, #redeem {padding-left: 15px}`

File: client/dist/style.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  margin-top: 8px;
+  margin-bottom: 15px;
+  width: 448px;
+  float:left;
+  height: 30;
+
+}
+
+.mybutton {
+  width: 33.3%;
```
Comment (willresen) [inline client/dist/style.css]: Is this button supposed to scale relative to the width of `window`?

File: client/dist/style.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+.description {
+  overflow: hidden;
+  max-height: 90px;
+  font-family: SystemFont,Helvetica Neue,Helvetica,Arial,sans-serif;
+  font-weight: 250;
+}
+
+.more_less {
+  float: right;
+}
```
Comment (willresen) [inline client/dist/style.css]: Make sure to leave a newline at the end of every file.

File: client/src/component/background.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import FilmInfo from './filminfo.jsx';
+
+const BackgroundImage = (props) => (
+  <div className="backgroundImage" style={
+    {backgroundImage: `url(${props.film.movie_shot_url})`}
+  }>
+    <FilmInfo film={props.film}/>
+    <div className="blueGradient"></div>
+  </div>
+)
```
Comment (willresen) [inline client/src/component/background.jsx]: Missing ;

File: client/src/component/filmStats.jsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  } else if(rating > 59){
+    return `url(https://www.vudu.com/bluesteel/images/Tomato_fresh.svg)`;
+  } else {
+    return `url(https://www.vudu.com/bluesteel/images/Tomato_rotten.svg)`;
+  }
+}
+
+let tomatoStyle={
+  'background-image': getTomatoStyle(90)
+}
```
Comment (willresen) [inline client/src/component/filmStats.jsx]: Several missing semi-colons in this file.

File: client/src/component/filmStats.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+// TODO host tomatos on s3
+let num = 3.5;
```
Comment (willresen) [inline client/src/component/filmStats.jsx]: What's this for?

File: client/src/index.jsx [hunk 7]
```diff
@@ -1,8 +1,41 @@
...
+  componentDidMount(){
+    console.log('fetch random film')
+    this.fetchRandFeaturedFilm();
+  }
+
+  fetchRandFeaturedFilm(){
+    fetch('/api/rand')
+    .then(res =>res.json())
+    .then((res) => {
+      console.log(res);
```
Comment (willresen) [inline client/src/index.jsx]: Make sure to remove any leftover `console.log`s before deploying.

File: package.json [hunk 8]
```diff
@@ -22,9 +22,12 @@
   "homepage": "https://github.com/fec1-arwen/FeaturedFilm#readme",
   "devDependencies": {
     "@babel/core": "^7.5.0",
+    "@babel/plugin-transform-runtime": "^7.5.0",
     "@babel/preset-env": "^7.5.2",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.6",
+    "enzyme": "^3.10.0",
```
Comment (willresen) [inline package.json]: Enzyme needs to be moved from dev to production dependencies.

File: server/index.js [hunk 9]
```diff
@@ -10,13 +10,18 @@ app.use(express.static(path.join(__dirname, '/../client/dist')));
 app.use(bodyparser.json());
 app.use(bodyparser.urlencoded({ extended: true }));
```
Comment (willresen) [inline server/index.js]: You should not need bodyparser here - there are no `POST` endpoints that read a request body.

File: tests/seeder.test.js [hunk 10]
```diff
@@ -1,4 +1,4 @@
-const seed = require('./seeder');
+const seed = require('../server/seeder');
 
 test('the movie list has given movie', async () => {
```
Comment (willresen) [inline tests/seeder.test.js]: Do you need the `async` keyword here?

File: client/dist/style.css (hunk 1 above)
Comment (HBMattoon) [reply client/dist/style.css]: yeah I was planning on going back and combining a ton of my styles. there is a lot of redundancy in there

File: client/dist/style.css (hunk 2 above)
Comment (HBMattoon) [reply client/dist/style.css]: its taking up one third of the containing div, which is set to 448px

Comment (willresen) [reply client/dist/style.css]: @HBMattoon AH I should’ve checked the component. Nice!

File: client/src/component/filmStats.jsx (hunk 6 above)
Comment (HBMattoon) [reply client/src/component/filmStats.jsx]: oops that's leftover from some early tests

**Pull request merged**